docs_src/environment_variables.md

```diff
@@ -0,0 +1,20 @@
+When running docker-run.sh script, we support environment variables as input for containers. Here is a list of environment variables and how you can apply it to docker-run.sh script as input
+
+| Environment Variable   | Purpose                                                                 |
```

Comment:
there are still some missing:  please also add `PIPELINE_PROFILE`, `STREAM_DENSITY_FPS`, and `STREAM_DENSITY_INCREMENTS` environment variables

Reply:
also the `RENDER_MODE`

Reply:
also the `DEVICE` env

Reply:
I noticed not all environment variables can be added to docker-run.sh script, I think these 3 should be fine. can you suggest purpose description for each and add them into #225

Reply:
could you do a draft descript first, and i can help you to edit it

Reply:
worked on other PR #228